music_assistant/providers/plex/manifest.json

```
@@ -3,7 +3,7 @@
   "domain": "plex",
   "stage": "unmaintained",
   "name": "Plex Media Server Library",
-  "description": "Support for the Plex streaming provider in Music Assistant.",
+  "description": "Stream your personal music, podcasts, and radio via your Plex media server.[Unmaintained]",
```

```suggestion
  "description": "Stream your personal music, podcasts, and radio via your Plex media server.",
```
Leave the stage flag from the description, I plan to use the stage property in the manifest to show some standardized label for that in the UI